cmd/peapod-to-fstree/main.go

```diff
@@ -0,0 +1,326 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"slices"
+	"strings"
+	"time"
+
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+	engineconfig "github.com/nspcc-dev/neofs-node/cmd/neofs-node/config/engine"
+	shardconfig "github.com/nspcc-dev/neofs-node/cmd/neofs-node/config/engine/shard"
+	fstreeconfig "github.com/nspcc-dev/neofs-node/cmd/neofs-node/config/engine/shard/blobstor/fstree"
+	objectcore "github.com/nspcc-dev/neofs-node/pkg/core/object"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/common"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/compression"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/fstree"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/peapod"
+	meta "github.com/nspcc-dev/neofs-node/pkg/local_object_storage/metabase"
+	"go.etcd.io/bbolt"
+	"gopkg.in/yaml.v3"
+)
+
+type epochState struct{}
+
+func (s epochState) CurrentEpoch() uint64 {
+	return 0
```

Comment:
Not sure what can be affected by this.

Reply:
How can I easily get the epoch state in another way? The metabase requires it to be set.

Reply:
Only via network interaction. Maybe it's not a problem in our case (updating storage ID doesn't require epoch IIRC). Or maybe we can get some latest one from meta or local state.

Reply:
I can get LastResyncEpoch from metabase:
https://github.com/nspcc-dev/neofs-node/blob/45802c65709a8be508ccd8b949efaf85933ddd6f/pkg/local_object_storage/metabase/last_resync_epoch.go#L12-L14

Reply:
It's somewhat different in meaning and it's a new one, 0.43.0 DB doesn't have it.

Reply:
why do you need a precise epoch? just pick an epoch that does not claim anything about object expiration and allows GET and PUT freely, that is `0`. @roman-khimov, what bothers you?

> How can I easily get the epoch state in another way? The metabase requires it to be set.

i did this and i did not know there would be so many utils that just open metabases and look at them. normal metabase always has to know the current epoch (via internal state or via explicit parameter to every operation)